airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/containers.py

```diff
@@ -34,6 +36,22 @@ def build_container(
     for key, value in env_vars.items():
         container = container.with_env_variable(key, value)
 
+    # Set the working directory to the code to format
+    container = container.with_workdir("/src")
+
+    # Mount a subset of the relevant parts of the repository, if requested.
+    # These should only be files which do not change very often.
+    # These can then be referenced by the install_commands.
+    if warmup_include:
+        container = container.with_mounted_directory(
```

Comment:
```suggestion
        container = container.with_directory(
```

I tested `with_directory` and `with_mounted_directory` with equivalent behavior/performance.
The difference is `with_directory` performs a COPY operation and `with_mounted_directory` performs a volume mount.
I tend to better trust `with_directory` in terms of caching behavior.

Reply:
The volume mount guarantees that the contents are overwritten in the subsequent mount, which I considered to be a desirable feature. I've added some commentary to explain this.